sso-kit-client/sso-kit-client-lit/src/index.ts

```diff
@@ -0,0 +1,20 @@
+/*-
+ * Copyright (C) 2022 Vaadin Ltd
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ *
+ * See <https://vaadin.com/commercial-license-and-service-terms> for the full
+ * license.
+ */
+export * from './SingleSignOnContext.js';
+export * from './AuthenticatedUser.js';
+
+const $wnd = window as any;
```

Comment:
You can avoid `any` and use `declare global` for `Vaadin.registrations` adding `is` and `version` properties.

Reply:
Removed `any`. Declared the types in global.